hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/AddUpdateRolloutWindowLayout.java

```diff
@@ -1106,18 +1106,11 @@ private void setAutoStartType(final Rollout rollout) {
 
     private enum ERROR_THRESHOLD_OPTIONS {
 
-        PERCENT("%"),
+        PERCENT,
+        COUNT;
 
-        COUNT("Count");
-
-        private final String value;
-
-        ERROR_THRESHOLD_OPTIONS(final String val) {
-            value = val;
-        }
-
-        private String getValue() {
-            return value;
+        private String getValue(final VaadinMessageSource i18n) {
+            return  i18n.getMessage("label.errorthreshold.option." + name().toLowerCase());
```

Comment:
Be careful here: Dynamic labels are hard to maintain. If you search e.g. for `label.errorthreshold.option.percent` your IDE won't find it hence you might conclude it's not used anywhere and remove it.

What about `PERCENT(label.errorthreshold.option.percent), COUNT(label.errorthreshold.option.count);` ?

Reply:
good point. like it.